src/postgraphql/__tests__/pathgraphqlIntegrationSchemaExport-test.js

```diff
@@ -0,0 +1,28 @@
+jest.mock('fs')
+
+import withPgClient from '../../postgres/__tests__/fixtures/withPgClient'
+import createPostGraphQLSchema from '../schema/createPostGraphQLSchema'
+import exportPostGraphQLSchema from '../schema/exportPostGraphQLSchema'
+import { writeFileSync } from 'fs'
+
+test('exports a schema as JSON', withPgClient(async pgClient => {
+  const gqlSchema = await createPostGraphQLSchema(pgClient, ['a', 'b', 'c'])
+  await exportPostGraphQLSchema(gqlSchema, { exportPathJSON: '/schema.json' })
```

Comment:
Never mind. I had to manually mock the `writeFile` method (as the callback needs to actually be called), and in doing this the tests now work 😄